### PR TITLE
various test fixes and generate test manifest

### PR DESCRIPTION
**Pull request by pmaria**

### CODE REVIEW

File: test-cases/RMLTC0000-SPARQL/mapping.ttl [hunk 1]
```diff
@@ -3,14 +3,13 @@
...
 <http://example.com/base#Country> rml:logicalSource [
-      rml:iterator "$.results.bindings[*]";
       rml:iterator """
             PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
             SELECT ?name
             WHERE {
                 ?name a rdfs:Literal .
             } """;
-      rml:referenceFormulation rml:JSONPath;
+      rml:referenceFormulation <https://www.w3.org/ns/formats/SPARQL_Results_JSON>;
```
Comment (DylanVanAssche) [inline test-cases/RMLTC0000-SPARQL/mapping.ttl:12]: Shouldn't we add the prefix above?

File: test-cases/RMLTC0015b-MySQL/mapping.ttl [hunk 2]
```diff
@@ -26,16 +26,16 @@
 <http://example.com/base/TriplesMap2> a rml:TriplesMap;
   rml:logicalSource [
       rml:iterator """
-            SELECT \"Code\", \"Name\", \"Lan\"
-            FROM \"Country\"
-            WHERE \"Lan\" = 'ES';
-           """;
+        SELECT Code, Name, Lan
```
Comment (DylanVanAssche) [inline test-cases/RMLTC0015b-MySQL/mapping.ttl:29]: Why changing this completely?

**Review by DylanVanAssche — CHANGES_REQUESTED** [review]

File: test-cases/RMLTC0015b-MySQL/mapping.ttl (hunk 2 above)
Comment (pmaria) [reply test-cases/RMLTC0015b-MySQL/mapping.ttl:29]: because it is invalid. The escaping is unnecessary in a multiline string

Comment (pmaria) [reply test-cases/RMLTC0015b-MySQL/mapping.ttl:29]: so this leads to an error, but not the expected error in the test case

**Review by DylanVanAssche — APPROVED** [review]

**Pull request merged**